packages/calcite-components/package.json

```diff
@@ -87,7 +87,7 @@
     "@stencil/state-tunnel": "1.0.1"
   },
   "license": "SEE LICENSE.md",
-  "volta": {
```

Comment:
I'm removing volta at this level because a calcite-components consumer will not have the context of the monorepo to reference the volta node version.

Reply:
The volta version is not for the calcite consumer. It is for automatically using the correct version of Node during development. Can you please add it back?

Reply:
If Volta adds another place to specify the development versions of node/npm then we can switch to that. But for now `package.json` is the only place to store pinned versions. We need to keep the `volta` field in CC's `package.json` so everyone who uses Volta will use the correct version of node. 

We are following the recommendation for monorepo setups in Volta's doc:
https://docs.volta.sh/advanced/workspaces

The `volta` field in our `package.json` has no impact on our consumers, even if they use Volta. They would need to pin a version on their end.

Can you add volta back here and then I'll approve the PR? Thanks!

Reply:
after another discussion with Ben over teams I'm adding volta back to calcite-components